app/containers/BalancesContainer/Token.js

```diff
@@ -0,0 +1,101 @@
+
+import BigNumber from 'bignumber.js';
+
+export class Token {
+
+	constructor(contractAddress, userAddress, symbol, decimal) {
+	    this.contractAddress = contractAddress;
+		this.userAddress = userAddress
+		this.symbol = symbol;
+		this.decimal = decimal;
+		this.balance = 0;
+		this.balanceHex = "0x70a08231";
+
+		this.unitMap = {
+			'wei': '1',
+			'kwei': '1000',
+			'ada': '1000',
+			'femtoether': '1000',
+			'mwei': '1000000',
+			'babbage': '1000000',
+			'picoether': '1000000',
+			'gwei': '1000000000',
+			'shannon': '1000000000',
+			'nanoether': '1000000000',
+			'nano': '1000000000',
+			'szabo': '1000000000000',
+			'microether': '1000000000000',
+			'micro': '1000000000000',
+			'finney': '1000000000000000',
+			'milliether': '1000000000000000',
+			'milli': '1000000000000000',
+			'ether': '1000000000000000000',
+			'kether': '1000000000000000000000',
+			'grand': '1000000000000000000000',
+			'einstein': '1000000000000000000000',
+			'mether': '1000000000000000000000000',
+			'gether': '1000000000000000000000000000',
+			'tether': '1000000000000000000000000000000'
+		};
+	}
+
+	prettyName() {
+		return this.symbol;
+	}
+
+	getDataObj(to, func, arrVals) {
+		var val="";
+	    for(var i=0;i<arrVals.length;i++) val+=this.padLeft(arrVals[i],64);
```

Comment:
A more JS-y way to write this is with a [`for...in` loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)